### PR TITLE
feat: add logging config file

### LINKED ISSUE #10: Set a logger for core app
Choose a convenient logger lib and use it in the python code

### PR DESCRIPTION
**Pull request by Oleksandr6676**
it adds logging config file

Closes #10, #36

### CODE REVIEW

**Comment by Oleksandr6676** [comment]
@JanMate I will add logic call and exception handlers in this PR as well, if you don't mind. It was easier to test logger on the same branch with implemented logic

**Comment by Oleksandr6676** [comment]
__name__ didn't work for me.  I defined 1 more logger in config file and used it in the transaction module instead of root, so we could keep track on errors and files

File: logging.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+propagate=0
+
+[handler_consoleHandler]
+class=StreamHandler
+level=INFO
+formatter=simpleFormatter
+args=()
+
+[formatter_simpleFormatter]
+format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
```
Comment (JanMate) [inline logging.conf:27]: Please, get rid of this conf file and replace it by happy_bank_core/config/log.py module that contains the following code:
```suggestion
import logging


def setup_logging():
    logging.basicConfig(
        level=logging.DEBUG,
        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
    )

```

That should be enough and unit tests still pass.

File: happy_bank_core/logic/transaction.py [hunk 2]
```diff
@@ -1,4 +1,7 @@
 from .account import Account
+import logging.config
+
+logger = logging.getLogger("transaction")
```
Comment (JanMate) [inline happy_bank_core/logic/transaction.py:4]:
```suggestion
logger = logging.getLogger(__name__)
```

File: happy_bank_core/logic/transaction.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
 from .account import Account
+import logging.config
```
Comment (JanMate) [inline happy_bank_core/logic/transaction.py:2]:
```suggestion
import logging
```

File: happy_bank_core/app.py [hunk 4]
```diff
@@ -3,22 +3,47 @@
 
 Represents REST Api layer.
 """
-
+import logging.config
```
Comment (JanMate) [inline happy_bank_core/app.py:6]:
```suggestion
import logging
```

File: happy_bank_core/app.py [hunk 5]
```diff
@@ -3,22 +3,47 @@
...
 from flask import Flask
```
Comment (JanMate) [inline happy_bank_core/app.py:7]:
```suggestion
from flask import Flask

```

File: happy_bank_core/app.py [hunk 6]
```diff
@@ -3,22 +3,47 @@
...
+from happy_bank_core.logic import account, transaction
+
+logging.config.fileConfig("../logging.conf")
```
Comment (JanMate) [inline happy_bank_core/app.py:10]:
```suggestion
```

File: happy_bank_core/app.py [hunk 7]
```diff
@@ -3,22 +3,47 @@
 
 Represents REST Api layer.
 """
-
+import logging.config
 from flask import Flask
+from happy_bank_core.logic import account, transaction
+
```
Comment (JanMate) [inline happy_bank_core/app.py:9]:
```suggestion
from happy_bank_core.config.log import setup_logging

```

File: happy_bank_core/app.py [hunk 8]
```diff
@@ -3,22 +3,47 @@
...
+logging.config.fileConfig("../logging.conf")
+logger = logging.getLogger()
```
Comment (JanMate) [inline happy_bank_core/app.py:11]:
```suggestion
setup_logging()
logger = logging.getLogger(__name__)
```

**Review by JanMate — CHANGES_REQUESTED** [review]
@Oleksandr6676, I took a look on that and my suggested changes can offer us to call `logger.info("...")` and in logs see the module name without much effort.

Check this out and let me know if it works properly for you and even in docker container.

**Review by JanMate — APPROVED** [review]

**Pull request merged**